src/jquery.contextMenu.js

```diff
@@ -1089,7 +1089,11 @@
                             $name.append(document.createTextNode(item._afterAccesskey));
                         }
                     } else {
-                        $name.text(item.name);
```

Comment:
Yea... I'd say that it would be a reasonable approach to "disable" the accessKey functionality altogether if the name is defined as being HTML.